src/components/ObsDetails/ObsDetails.js

```diff
@@ -293,28 +351,37 @@ const ObsDetails = (): Node => {
           <PhotoScroll photos={photos} />
           {/* TODO: a11y props are not passed down into this 3.party */}
           <IconButton
-            icon={currentUserFaved ? "star-bold-outline" : "pencil"}
-            onPress={faveOrUnfave}
+            icon="pencil"
             textColor={colors.white}
-            className="absolute top-3 right-0"
+            className="absolute top-3 right-3"
             accessible
             accessibilityRole="button"
-            accessibilityLabel={
-              currentUserFaved
-                ? t( "Fave-button-label-unfave" )
-                : t( "Fave-button-label-fave" )
-            }
+            accessibilityLabel={t( "edit" )}
           />
+          <IconButton
+            icon="star-bold-outline"
+            size={25}
+            onPress={() => faveOrUnfave()}
+            textColor={colors.white}
+            className="absolute bottom-3 right-3"
+            accessible
+            accessibilityRole="button"
+            accessibilityLabel={t( "favorite" )}
+          />
+          <View className="absolute bottom-3 left-3">
+            <PhotoCount count={photos?.length} />
```

Comment:
The `count` prop has to be a number, so either give it a default of 0 here or change the component to accept `undefined`.